docs/workforce-management-framework/topics/con-logging.adoc

```diff
@@ -1,10 +1,16 @@
 [id='{context}-con-logging']
 
-*{PROJECT_NAME} Logging* is a facade for link:https://nodejs.org[Node.js] based applications and it is used by all {PROJECT_NAME} modules.
-*{PROJECT_NAME} Logging* is contained in the `{PROJECT_NAME}-core` module.
+*{PROJECT_NAME} Logger* creates logging facade for JavaScript based applications that
+is used by all RainCatcher modules. Developers can use their own loggers by wrapping their implementation into the provided logger interface.
```

Comment:
>that
 -*{PROJECT_NAME} Logging* is contained in the `{PROJECT_NAME}-core` module.		 +is used by all RainCatcher modules.

suggestion: '... aplications, it is used by all RAin..'

Reply:
>Developers can use their own loggers by wrapping their implementation into the provided logger interface.

Developers can use custom loggers by wrapping their implementation into the provided logger interface.

"their" is repeated twice in one sentence, reads bit awkward.